steps/github-step-repositories-describe/step.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+from github import Github, UnknownObjectException
+from relay_sdk import Interface, Dynamic as D
+
+relay = Interface()
+
+try:
+  token = relay.get(D.token)
+  org = relay.get(D.org)
+except:
+  raise ValueError("Token and org are required parameters")
+
+scope = relay.get(D.scope)
+sort = relay.get(D.sort)
+direction = relay.get(D.direction)
```

Comment:
Unfortunately the defaults from the schema won't come through if someone writes the step spec by hand, so you need to guard anything that isn't required, e.g.:

```python
try:
  scope = relay.get(D.scope)
except:
  scope = 'all'
```